Dockerfile

```
@@ -24,6 +24,9 @@ RUN echo "{\"sha\": \"$(git rev-parse HEAD)\", \"version\": \"$(git describe --t
 FROM docker.io/nginx:1.25
 RUN mkdir /etc/nginx/templates
 COPY dockerconf/nginx.conf.template /etc/nginx/templates/default.conf.template
-COPY --from=build-stage /app/dist /usr/share/nginx/html
+COPY --from=build-stage /app/dist /usr/share/nginx/html/frontend
+# second index.html needed for the location /h5p/ in csp rules
+COPY --from=build-stage /app/dist /usr/share/nginx/html/h5p
+
```

shoud be work with the old line
It not necessary to copy the same files two times
```suggestion
COPY --from=build-stage /app/dist /usr/share/nginx/html
```

---

If both locations have the same root path, bugs occur when loading the index.html in try_files.